From 594c1e87aaac1513457dafc0054295cd1c8c7e5a Mon Sep 17 00:00:00 2001
From: Stephano Cetola <stephano.cetola@linux.intel.com>
Date: Wed, 5 Apr 2017 10:09:08 -0700
Subject: [PATCH] calibrate: Add printk example

Add some printk stuff to the kernel as an example for testing.

Signed-off-by: Stephano Cetola <stephano.cetola@linux.intel.com>
---
 init/calibrate.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/init/calibrate.c b/init/calibrate.c
index ce635dccf3d9..0bd04c7c5359 100644
--- a/init/calibrate.c
+++ b/init/calibrate.c
@@ -277,6 +277,12 @@ void calibrate_delay(void)
 	static bool printed;
 	int this_cpu = smp_processor_id();
 
+	printk("*************************************\n");
+	printk("*                                   *\n");
+	printk("*        HELLO YOCTO KERNEL         *\n");
+	printk("*                                   *\n");
+	printk("*************************************\n");
+
 	if (per_cpu(cpu_loops_per_jiffy, this_cpu)) {
 		lpj = per_cpu(cpu_loops_per_jiffy, this_cpu);
 		if (!printed)
-- 
2.12.2

